script.js

```diff
@@ -0,0 +1,115 @@
+// 1. What is a function?
+
+// ფუნქცია არის კოდის ბლოკი, რომელიც შექმნილია კონკრეტული ამოცანის შესასრულებლად და შეიძლება შესრულდეს გამოძახებისას. ფუნქციები ხელს უწყობს კოდის ორგანიზებას, 
```

Comment:
აუცილებლად შესრულდება გამოძახებისას!
```suggestion
// ფუნქცია არის კოდის ბლოკი, რომელიც შექმნილია კონკრეტული ამოცანის შესასრულებლად და შესრულდება გამოძახებისას. ფუნქციები ხელს უწყობს კოდის ორგანიზებას, 
```